portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="bulma.min.css" />
+        <link rel="stylesheet" href="style.css" />
+        <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="mapScript.js"></script>
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBcC9VE5rKnzGtbd4bHlHUeOeeS6N5ajTA"></script>
```

Comment:
So I would avoid putting your API key in a git commit. Anyone browsing your code could then use your API key, and its use would be attributed to you. (Making a new commit to remove it would technically not fix the problem since the previous commit that contained it would still exist.)